manual/mapper/mapper/README.md

```diff
@@ -36,6 +36,22 @@ CqlSession session = CqlSession.builder().build();
 InventoryMapper inventoryMapper = new InventoryMapperBuilder(session).build();
 ```
 
+One nice trick you can use is to create a static factory method on your interface. This hides the
+name of the generated class from the rest of your application:
+
+```java
+@Mapper
+public interface InventoryMapper {
+  
+  static MapperBuilder<InventoryMapper> builder(CqlSession session) {
+    return new InventoryMapperBuilder(session);
+  }
+  ...
+}
+
+InventoryMapper inventoryMapper = InventoryMapper.builder(session).build();
+```
+
```

Comment:
Not strictly related to this PR, but I just realized you can do this and I think it's pretty cool.